src/commands/suggestions/AddSuggestion.ts

```diff
@@ -2,6 +2,12 @@ import {Command, CommandoClient, CommandoMessage} from "discord.js-commando";
 import {Message} from "discord.js";
 import {SuggestionsManager} from "../../utility/SuggestionsManager";
 
+/**
+ * Enum for potential results
+ * @constant
+ */
+const resultEnum = {LISTEXISTS: 6, SUGGESTIONDOESNOTEXIST: 5, LISTADDED: 4, DERP: 3, SUGGESTIONADDED: 2, LISTDOESNOTEXIST: 1, SUGGESTIONEXISTS: 0};
```

Comment:
Also suggest defining enums as enum type.

Additionally, suggest putting and exporting this enum type in the associated class that is actually generating the result codes rather than having to redefine the type outside. This, combined with the enum typing will allow said function returning the status return the enum type rather than a regular "number" allowing for increased type safety.

Reply:
Turns out that to export it, you have to use "export" good to know xD

Reply:
scratch that. Enum is now internal to the SuggestionManager class, as it should be. Huzzah